cvat-ui/src/config.tsx

```diff
@@ -111,7 +111,7 @@ const HEALTH_CHECK_RETRIES = 10;
 const HEALTH_CHECK_PERIOD = 3000; // ms
 const HEALTH_CHECK_REQUEST_TIMEOUT = 5000; // ms
 
-const CONTROLS_LOGS_INTERVAL = 90000; // 1.5 min in ms
+const CONTROLS_LOGS_INTERVAL = 180000; // 3 min
```

Comment:
Do we really want to change it? If so, probably some changes on server side are needed.
It can affect working time count logic [here](https://github.com/opencv/cvat/blob/develop/cvat/apps/events/serializers.py#L33).

Reply:
I don't think it's need to change the server, my concern is that more events may not be sent because of closing tab.

Reply:
okay, let's revert